graspy/utils/utils.py

```diff
@@ -38,14 +38,23 @@ def import_graph(graph):
     if isinstance(graph, (nx.Graph, nx.DiGraph, nx.MultiGraph, nx.MultiDiGraph)):
         out = nx.to_numpy_array(graph, nodelist=sorted(graph.nodes), dtype=np.float)
     elif isinstance(graph, (np.ndarray, np.memmap)):
-        maximum = np.max(graph.shape)
+        shape = graph.shape
+        if len(shape) == 3:
```

Comment:
is there any check for len(shape) > 4? If not I think line 49 could break if there is a 4+d array and one of the dimensions is really long